### ISSUE TITLE
Add spinner to actions button

### ISSUE THREAD

**Issue by taylortom** [adapt-security/adapt-authoring-ui#223]
### Related to no-sidebar branch

***

Actions buttons should support an animation to show async tasks are running.

**Comment by chris-steele** [comment]
Yep we need to show the user some indication/feedback whenever something is being worked on server-side
